Google.PowerShell/Compute/GceManagedInstanceGroupCmdlets.cs

```diff
@@ -1335,5 +1365,25 @@ protected override void ProcessRecord()
                 instances = request.Execute().ManagedInstances;
             }
         }
+
+        private void WaitRegionManagedInstance(string project, string region, string name)
+        {
+            RegionInstanceGroupManagersResource.ListManagedInstancesRequest request =
+                Service.RegionInstanceGroupManagers.ListManagedInstances(project, region, name);
+
+            Stopwatch stopwatch = Stopwatch.StartNew();
+            IList<ManagedInstance> instances = request.Execute().ManagedInstances;
+            while (instances != null && instances.Any(i => i.CurrentAction != "NONE") && !Stopping)
+            {
+                if (Timeout >= 0 && stopwatch.Elapsed.Seconds > Timeout)
+                {
+                    WriteWarning("Wait-GceManagedInstanceGroup timed out for " +
+                                 $"{request.Project}/{request.Region}/{request.InstanceGroupManager}");
+                    break;
+                }
+                Thread.Sleep(150);
```

Comment:
No need to address here but should we make this configurable?

Reply:
I think we can either add another parameter for that or set a config for the whole module. Nobody has asked for this before so I'm not sure how useful it is though.